documentation/fundamentals/requirements.qmd

```diff
@@ -424,3 +424,21 @@ nextflow run hello -with-docker
     Hello world!
     Hola world!
     Ciao world!
+
+
+## Optional: install VS Code
+It is recommended to install and use VS Code as this will help with the development of components due to the use of extensions and YAML schema's.
```

Comment:
```suggestion
It is recommended to install and use VS Code as this will help with the development of components due to the use of extensions and YAML schemas.
```